integration-tests/docker/test_env/test_env.go

```diff
@@ -59,15 +59,23 @@ func NewTestEnv() (*CLClusterTestEnv, error) {
 	if err != nil {
 		return nil, err
 	}
-	networks := []string{network.Name}
+	n := []string{network.Name}
 	return &CLClusterTestEnv{
+		Geth:       test_env.NewGeth(n),
+		MockServer: test_env.NewMockServer(n),
 		Network:    network,
-		Geth:       test_env.NewGeth(networks),
-		MockServer: test_env.NewMockServer(networks),
 		l:          log.Logger,
 	}, nil
 }
 
+func (te *CLClusterTestEnv) WithTestEnvConfig(cfg *TestEnvConfig) *CLClusterTestEnv {
```

Comment:
@tateexon could you add a comment for this function? This would hopefully make it easier to reuse in other projects (like Mercury)

Reply:
Comment added